--- DSA.xs.orig	2009-08-05 12:18:29.000000000 +0200
+++ DSA.xs	2009-08-05 12:19:04.000000000 +0200
@@ -54,7 +54,7 @@
         }
         dsa = DSA_generate_parameters(bits, seedpv, seed_len, NULL, NULL, NULL, NULL);
         if (!dsa)
-          croak(ERR_reason_error_string(ERR_get_error()));
+          croak("%s", ERR_reason_error_string(ERR_get_error()));
         RETVAL = dsa;
     OUTPUT:
         RETVAL
